ios/Plugin/CapacitorCalendarPlugin.swift

```diff
@@ -440,6 +512,38 @@ public class CapacitorCalendarPlugin: CAPPlugin {
         }
     }
 
+    @objc public func deleteEventById(_ call: CAPPluginCall)
```

Comment:
This method is not registered in `CapacitorCalendarPlugin.m`.

Reply:
Should now be registered.